### ISSUE TITLE
Can't insert some models (Atlas) from the Gazebo GUI

### ISSUE THREAD

**Issue by osrf-migration** [osrf/drcsim#405]
**Original report ([archived issue](https://osrf-migration.github.io/drcsim-gh-pages/#!/osrf/drcsim/issues/405)) by Hugo Boyer (Bitbucket: [hugomatic](https://bitbucket.org/%7B3d87874e-7099-4981-b04f-57f0272faa7d%7D/), GitHub: [hugomatic](https://github.com/hugomatic)).**

The original report had attachments: [Gazebo : default_001.png](https://osrf-migration.github.io/drcsim-gh-pages/data/repositories/osrf/drcsim/issues/405/attachments/Gazebo__default_001.png)

----------------------------------------

My model path points to a directory that conains models. However, Gazebo (version 3.0.0) does not allow me to insert them. 

```
#!bash
GAZEBO_MODEL_PATH=/tmp/ws/install/share/drcsim_model_resources/gazebo_models/environments:/tmp/ws/install/share:
```

Where /tmp/ws/install/share contains:

```
#!bash

drwxrwxr-x 10 hugo hugo 4096 Nov 27 10:36 atlas_description/
drwxrwxr-x  6 hugo hugo 4096 Nov 27 10:36 atlas_msgs/
drwxrwxr-x  3 hugo hugo 4096 Nov 27 10:36 common-lisp/
drwxrwxr-x  2 hugo hugo 4096 Nov 27 10:36 drcsim/
drwxrwxr-x  8 hugo hugo 4096 Nov 27 10:36 drcsim_gazebo/
drwxrwxr-x  3 hugo hugo 4096 Nov 27 10:36 drcsim_gazebo_plugins/
drwxrwxr-x  3 hugo hugo 4096 Nov 27 10:36 drcsim_gazebo_ros_plugins/
drwxrwxr-x  8 hugo hugo 4096 Nov 27 10:55 drcsim_model_resources/
drwxrwxr-x  5 hugo hugo 4096 Nov 27 10:36 gazebo_msgs/
drwxrwxr-x  5 hugo hugo 4096 Nov 27 10:36 gazebo_plugins/
drwxrwxr-x  4 hugo hugo 4096 Nov 27 10:36 gazebo_ros/
drwxrwxr-x  2 hugo hugo 4096 Nov 27 10:36 gazebo_ros_pkgs/
drwxrwxr-x  8 hugo hugo 4096 Nov 27 10:36 irobot_hand_description/
drwxrwxr-x  8 hugo hugo 4096 Nov 27 10:36 multisense_sl_description/
drwxrwxr-x  4 hugo hugo 4096 Nov 27 10:36 osrf_msgs/
drwxrwxr-x  8 hugo hugo 4096 Nov 27 10:36 sandia_hand_description/
drwxrwxr-x  3 hugo hugo 4096 Nov 27 10:36 sandia_hand_driver/
drwxrwxr-x  5 hugo hugo 4096 Nov 27 10:36 sandia_hand_msgs/
drwxrwxr-x  3 hugo hugo 4096 Nov 27 10:36 sandia_hand_teleop/

```

**Comment by osrf-migration** [comment]
**Original comment by Steve Peters (Bitbucket: [Steven Peters](https://bitbucket.org/%7B2ccfed09-18b8-4921-8d58-15ef01092802%7D/), GitHub: [scpeters](https://github.com/scpeters)).**

----------------------------------------
Can you elaborate on the problem? Were there any console error messages? Were you able to insert models from the online model database?

**Comment by osrf-migration** [comment]
**Original comment by John Hsu (Bitbucket: [hsu](https://bitbucket.org/%7B0a186eae-abf0-4514-a951-23db5eccc286%7D/), GitHub: [hsu](https://github.com/hsu)).**

----------------------------------------
Is this addressed by [pull request #408](https://osrf-migration.github.io/drcsim-gh-pages/#!/osrf/drcsim/pull-requests/408)?  Should we back port it into drcsim_3.1 and/or drcsim_3.0 branches?

**Comment by osrf-migration** [comment]
**Original comment by Nate Koenig (Bitbucket: [Nathan Koenig](https://bitbucket.org/%7Bc862cdd9-fcc8-4419-9fc7-e20db14b8fcb%7D/)).**

----------------------------------------

* changed state from "new" to "resolved"

Resolved in [pull request #408](https://osrf-migration.github.io/drcsim-gh-pages/#!/osrf/drcsim/pull-requests/408)
